pages/soundtrack/soundtrack.pixy

```diff
@@ -13,44 +13,43 @@ component Track(track *arn.SoundTrack)
 					h3.widget-title= media.Service
 					.sound-track-media
 						ExternalMedia(media)
-			
+
 			.widget.mountable
 				h3.widget-title Anime
 
 				.sound-track-anime-list
 					each anime in track.Anime()
 						a.sound-track-anime-list-item.ajax(href=anime.Link(), title=anime.Title.Canonical)
 							img.sound-track-anime-list-item-image.lazy(data-src=anime.Image.Tiny, alt=anime.Title.Canonical)
-			
+
 			if len(track.Beatmaps()) > 0
 				.widget.mountable
 					h3.widget-title Beatmaps
 					ul.beatmaps
 						for index, beatmap := range track.Beatmaps()
 							li
 								a.beatmap(href="https://osu.ppy.sh/s/" + beatmap, target="_blank")= "Beatmap #" + strconv.Itoa(index + 1)
-			
 			.widget.mountable
 				h3.widget-title Tags
 				.tags
 					each tag in track.Tags
 						.tag= tag
-
 		.footer.text-center.mountable
 			if track.EditedBy != ""
-				span Edited 
+				span Edited
```

Comment:
You can't remove whitespaces in Pixy files. Configure your editor to not delete whitespaces in .pixy files.